retailer-oas/retailer.yaml

```diff
@@ -823,9 +806,10 @@ components:
           type: string
           description: The unique identifier of the Order
           example: "order1"
-        timeSlot:
-          $ref: '#/components/schemas/TimeSlot'
-          description: The Time Slot selected for the Order
+        timeSlotId:
+          type: string
+          description: The Timeslot's id selected for the Order
```

Comment:
Nit: use `TimeSlot's identifier` instead of `Timeslot's id` to keep writing consistent